Program.fs

```diff
@@ -129,7 +118,7 @@ module MainModule =
     let sendEmailMessage (tally: seq<EmailTally>) = async {
         let base64HeaderValue = sprintf "%s:" swuKey |> Text.Encoding.UTF8.GetBytes |> Convert.ToBase64String
         let header = Http.Headers.AuthenticationHeaderValue ("Basic", base64HeaderValue)
-        let url = sprintf "https://api.sendwithus.com/api/v1/send"
+        let url = "https://api.sendwithus.com/api/v1/send"
```

Comment:
Good catch! I think I was using the `sprintf` to format part of the url there previously but it became unnecessary. Must have forgot to remove the `sprintf`.